docs/guides/cloud/debugging/test-replay.mdx

```diff
@@ -0,0 +1,170 @@
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/command-log.png"
+  alt="Test Replay Developer Tools panel"
+  width="60%"
+/>
+
+### Developer Tools
+
+Dive into network-level traffic, console events and inspect the application under test just as you can in the browser.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/dev-tools.png"
+  alt="Test Replay Developer Tools panel"
+  width="60%"
+/>
+
+### Replay Controls
+
+Toggle each test attempt, play back errors at a variety of speeds or scrub through the captured error by dragging the playhead along the timeline.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/scrubber.png"
+  alt="Test Replay scrubber controls"
+/>
+
+## Opt out of Test Replay
+
+:::info
+
+Please refer to the [Cypress Cloud Terms of Use](https://cloud.cypress.io/terms-of-use) and our [Security & Compliance](https://www.cypress.io/security/) guide for specifics around reasonable use of Cypress Cloud, data storage and security at Cypress.
+
+Test Replays, including the test data leveraged within your tests, are visible to everyone with access to your project.
+
+:::
+
+If you do not want to capture test data for replay and debugging purposes simply "shut off" Test Replay in project settings.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/opt-out.png"
+  alt="Test Replay opt-out controls"
+/>
+
+## Troubleshooting
+
+**If Test Replay is not present or disabled in Cypress Cloud:**
+
+1. Confirm runs have been recorded to Cypress Cloud using Cypress `v13` or later.
+2. Ensure tests were recorded using a Chromium-based browser (Chrome, Edge, Electron).
+3. Check that Test Replay is enabled in Cypress Cloud project settings.
+4. Review the standard output of the test run to ensure there was not an error capturing Test Replay. If you encounter an error capturing Test Replay, please open an issue so that we can investigate.
+
+## Command Log Notes
+
+Cypress uses a powerful [command log](/guides/core-concepts/cypress-app#Command-Log) locally to display every command executed in your spec. The Cypress command log is visible during `cypress open` mode, but is also rendered ([headless](/guides/guides/command-line#Options)-ly) during `cypress run` so that it displays within captured screenshots or videos, to assist in debugging CI failures.
+
+### Problem
+
+Rendering the Cypress command log while running tests via `cypress run` often has performance implications, resulting in a slower test runtime, especially on lower resourced machines.
+
+### Solution
+
+With Test Replay, Cypress regenerates the entire UI within the Cloud for viewing the run and the debugging CI failures. Additionally, displaying the Cypress command log in videos and screenshots is less valuable for users using Test Replay.
+
+Because of this, as of Cypress `v13`, the Cypress command log is no longer rendered during `cypress run` for users using Test Replay, with an option to opt out of this behavior.
```

Comment:
```suggestion
Because of this, as of Cypress `v13`, the Cypress command log is no longer rendered during `cypress run` for users using Test Replay, with an option to opt-out of this behavior.
```